### PR TITLE
mgr/dashboard: Verify fields on Monitors page

### PR DESCRIPTION
**Pull request by rafaelquint**
Fixes: https://tracker.ceph.com/issues/41048

Signed-off-by: Adam King <adking@redhat.com>
Signed-off-by: Rafael Quintero <rquinter@redhat.com>

### CODE REVIEW

**Comment by rafaelquint** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.e2e-spec.ts [hunk 1]
```diff
@@ -20,4 +21,95 @@ describe('Monitors page', () => {
       expect(monitors.getBreadcrumbText()).toEqual('Monitors');
     });
   });
+  describe('fields check', () => {
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.e2e-spec.ts]:
```suggestion

  describe('fields check', () => {
```

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.po.ts [hunk 2]
```diff
@@ -1,5 +1,18 @@
+import { $$ } from 'protractor';
 import { PageHelper } from '../page-helper.po';
 
 export class MonitorsPageHelper extends PageHelper {
   pages = { index: '/#/monitor' };
+
+  getStatusTable() {
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.po.ts]: All 3 new methods are not specific to the monitor page, so I think they should be moved to the `page-helper.po.ts` file.

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.e2e-spec.ts [hunk 3]
```diff
@@ -20,4 +21,95 @@ describe('Monitors page', () => {
...
+      expect(monitors.getStatusTable().getText()).toMatch('monmap epoch');
+      expect(monitors.getStatusTable().getText()).toMatch('quorum con');
+      expect(monitors.getStatusTable().getText()).toMatch('quorum mon');
+      expect(monitors.getStatusTable().getText()).toMatch('required con');
+      expect(monitors.getStatusTable().getText()).toMatch('required mon');
+    });
+
+    it('should check In Quorum and Not In Quorum tables are present', () => {
+      // check for there to be two tables
+      expect($$('cd-table').count()).toEqual(2);
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.e2e-spec.ts]: `$$('cd-table')` should be extracted to a method on `page-helper.po.ts`. [info](http://www.protractortest.org/#/style-guide#page-objects)

**Comment by rafaelquint** [comment]
Jenkins test dashboard

**Comment by rafaelquint** [comment]
Jenkins test make check

**Pull request merged**